### ISSUE TITLE
Robyn App - Exporting 1-Pagers

### ISSUE THREAD

**Issue by F1nalFortune** [facebookexperimental/Robyn#503]
## Describe issue
Currently am facing an issue when running the Robyn App with pre-loading the sample data. The error arises when plotting one-pagers into output directory.

```
> Exporting pareto results as CSVs into directory...
>> Exporting general plots into directory...
>>> Calculating clusters for model selection using Pareto fronts...
>> Auto selected k = 9 (clusters) based on minimum WSS variance of 5%
>>> Exporting pareto one-pagers into directory...
>> Generating only cluster results one-pagers (9)...
>> Plotting 9 selected models on 3 cores...
  |                                                                                                 |   0%Failed exporting results, but returned model results anyways:
 Error in {: task 1 failed - "could not find function "scale_y_percent""
```

## Environment & Robyn version
Robyn_3.7.3
RobynLearn_1.0.0
R version 4.1.2

**Comment by laresbernardo** [comment]
Hi! Can you please share what version of `lares` package you have installed? Can you try running `?lares::scale_y_percent` and see if the documentation is available? As you can see [here](https://github.com/facebookexperimental/Robyn/blob/main/R/R/imports.R#L32), the function is imported correctly by `Robyn`. Maybe try refreshing you session and trying again to see if there's some cache issue there.

**Comment by F1nalFortune** [comment]
After refreshing the session and re-installing Robyn and RobynLearn, I had to first import Robyn and then import RobynLearn, otherwise RobynLearn was loading "dev-3.5.0" and inherently not setting correct iterations per trial.

I ran `?lares::scale_y_percent` and did see the documentation - although I am still receiving same error.

**Comment by laresbernardo** [comment]
Please, uninstall Robyn 3.5.0 from your machine and try again. You might have an issue with different versions. You should load a newer version for RobynLearn to work correctly (say +3.7.0)

**Comment by F1nalFortune** [comment]
Thank you very much! Issue resolved after removing version 3.5 from machine.
